app/jobs/animals_job.rb

```diff
@@ -0,0 +1,40 @@
+class AnimalsJob < ApplicationJob
```

Comment:
Do we want to be a little more specific with the name of this job? Maybe something like `ImportAnimalsJob`?

Reply:
Good call